shell/platform/common/accessibility_bridge.cc

```diff
@@ -111,6 +111,23 @@ AccessibilityBridge::GetPendingEvents() {
   return result;
 }
 
+void AccessibilityBridge::UpdateDelegate(
+    std::unique_ptr<AccessibilityBridgeDelegate> delegate) {
+  delegate_ = std::move(delegate);
+  // FlutterPlatformNodeDelegate[s] are created by the
+  // AccessibilityBridgeDelegate and may contain states from previous
+  // AccessibilityBridgeDelegate. Need to recreate them.
+  for (const auto& pair : id_wrapper_map_) {
```

Comment:
Since you're using both key and value, consider using a structured binding for pair:
```cc
for (const auto& [node_id, flutter_node_delegate] : id_wrapper_map_) {
```
The explicit names help with readability vs first/second.